tools/osx-pkg-preinstall.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# TODO Can this be done inside the .pmdoc?
+# TODO Can we extract $PREFIX from the installer?
+cd /usr/local/bin
+rm -rf ../lib/node_modules/npm
```

Comment:
wow, good question. That is how the postinstall script is done. And now that I look again, it was done that way for a reason. Updating now